Docker/ProxyNginx.conf

```diff
@@ -0,0 +1,41 @@
+upstream dash-handler {
+  server 127.0.0.1:9002;
+}
+
+server {
+  listen 80;
+  server_name dash.example.com;
```

Comment:
Ah, I see that this is just an example, @SolidNerd why you don't want to include nginx in this image?